src/components/EndSession.jsx

```diff
@@ -104,16 +91,47 @@ const Button = styled.button`
 `;
 
 function EndSession() {
+    const emailRef = useRef();
+    const sessionCode = sessionStorage.getItem("sessionCode");
+
+    useEffect(() => emailjs.init("OScF2lHq5ESl_o9lU"), []);
+    const sendEmail = async (e) => {
+        e.preventDefault();
+        const serviceId = "service_2ndgu8t";
+        const templateId = "template_feo851p";
+
+        let transcriptUrl = 'http://localhost:8080/session/transcript/' + sessionCode
+
+        try {
+            const response = await axios.get(transcriptUrl);
+            console.log(response);
+            
+            if (!response || !response.data) {
+                    throw new Error('Invalid response from the server');
+                  }
+
+            const transcript = response.data;
+            console.log('Transcript:', transcript);
+
+            await emailjs.send(serviceId, templateId, {
+                recipient: emailRef.current.value,
+                message: transcript
+            });
+            alert("Email successfully sent. Check inbox.");
+        } catch (error) {
+            alert("Oops! Something went wrong while trying to send the email. Please make sure there are messages in the conversation before sending.");

```

Comment:
Can you change the alerts to mui alerts to stay consistent with the ones in my pull request? 
- This is the documentation: https://mui.com/material-ui/react-alert/
- You can reference my pr to see how to make them disappear automatically after a certain amount of time so users don't need to close it